File: apps/site/snippets/perf/reduced-motion.swift
import SwiftUI
import Sinua

struct QuietOrb: View {
    var body: some View {
        // .auto (default) follows Settings > Accessibility > Reduce Motion: a still pose.
        // With a voice attached it still redraws (up to 30 Hz): the voice cue is information.
        SinuaOrb(pattern: .working, reducedMotion: .auto)
            .frame(width: 160, height: 160)
    }
}

File: apps/site/snippets/perf/reduced-motion.kt
package snippets.perf

import androidx.compose.foundation.layout.size
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.unit.dp
import dev.sinua.view.FxReducedMotion
import dev.sinua.view.generated.SinuaOrb
import dev.sinua.view.generated.SinuaOrbPattern

@Composable
fun QuietOrb() {
    // AUTO (default) follows "Remove animations": a still pose.
    // With a voice attached it still redraws (up to 30 Hz): the voice cue is information.
    SinuaOrb(pattern = SinuaOrbPattern.WORKING, reducedMotion = FxReducedMotion.AUTO, modifier = Modifier.size(160.dp))
}
